src/Routes.js

```diff
@@ -77,7 +77,7 @@ import { routes } from "./constants/routes";
 
 const Routes = () => {
   return (
-    <BrowserRouter>
+    <BrowserRouter basename={process.env.PUBLIC_URL}>
```

Comment:
Will this work if I do not have `PUBLIC_URL` defined in my env?

Reply:
If we add `PUBLIC_URL` to the env then our localhost will also run on `/FOSSlogyUI`, and on gh-pages we are adding public url so at the time of deployment it will take FOSSologyUI from that env.

Reply:
That I understood but for normal user, `PUBLIC_URL` is not defined when they do `yarn build`. What will happen in that case?
And what if I added `homepage` to `package.json` instead?

Reply:
Yeah, I can add `"homepage" : "/FOSSologyUI"` to package.json but in that case our locahost will also run on `http://localhost:3000/FOSSologyUI`

Reply:
Let me try checking it on any sample repo. That how we can configure it only for production, but probably this should work for gh-pages.

Reply:
I didn't meant to say "use `homepage` from `package.json`", but rather what will happen if I do that instead of `export PUBLIC_URL=""`. But I just tested it, it works in both cases as expected.